source/localizable/models/finding-records.md

```diff
@@ -69,22 +69,42 @@ this.get('store').query('person', {
 
 ### Querying for A Single Record
 
-If you know your query will return only one result Ember Data provides
-a convenience method that will return a promise that resolves with a
-single record. Calling
-[`store.queryRecord()`](http://emberjs.com/api/data/classes/DS.Store.html#method_queryRecord)
-will make a `GET` request with the passed object serialized as query params.
+If you are using an adapter that supports server requests capable of returning a single model object, Ember Data provides
+a convenience method [`store.queryRecord()`](http://emberjs.com/api/data/classes/DS.Store.html#method_queryRecord) that will return a promise that resolves with that single record. The request is made via a method of the same name defined on the adapter.
 
-For example, if we know that an email uniquely identifies a person, we could search for a `person` model that has an email address of
-`tomster@example.com`:
+For example, if your server API provides an endpoint for the currently logged in user:
 
-```javascript
-// GET to /persons?filter[email]=tomster@example.com
-this.get('store').queryRecord('person', {
-  filter: {
-    email: 'tomster@example.com'
+```text
+// GET /api/current_user
+{
+  user: {
+    id: 1234,
+    username: 'admin'
+  }
+}
+```
+and the adapter for the `User` model defines a `queryRecord()` method that targets that endpoint:
+
+```app/adapters/user.js
+// app/adapters/user.js
+import DS from "ember-data";
+
+export default DS.Adapter.extend({
+  queryRecord(modelName, query) {
+    return Ember.$.getJSON("/api/current_user");
   }
-}).then(function(tomster) {
-  // do something with `tomster`
 });
 ```
+then calling
+[`store.queryRecord()`](http://emberjs.com/api/data/classes/DS.Store.html#method_queryRecord)
+will retrieve that object from the server:
+
+```javascript
+store.queryRecord('user', {}).then(function(user) {
+  let username = user.get('username');
+  console.log(`Currently logged in as ${username}`);
+});
+```
+As in the case of `store.query()`, a query object can also be passed to `store.queryRecord()` and is available for the adapter's `queryRecord()` to use to qualify the request. However the adapter must return a single model object, not an array containing one element, otherwise Ember Data will throw an exception.
```

Comment:
I think it might be good to add a paragraph explaining that if the API returns an array for the endpoint (probably with only 1 element), then one could re-use the `query` behavior and do:

```js
store.query("user", {}).then(function(users) {
  return users.get("firstObject");
});
```

---

We probably don't want to advertise the usage of addons in the guide, but just for the sake of completeness, there would be one available https://emberobserver.com/addons/ember-data-query-first...